broker/src/main/java/io/zeebe/broker/system/partitions/ZeebePartition.java

```diff
@@ -84,7 +84,7 @@
   private final TypedRecordProcessorsFactory typedRecordProcessorsFactory;
   private final CommandApiService commandApiService;
   private final List<PartitionListener> partitionListeners;
-  private final List<Actor> closeables = new ArrayList<>();
+  private final List<AsyncClosable> asyncClosables = new ArrayList<>();
```

Comment:
```suggestion
  private final List<AsyncClosable> managedResources = new ArrayList<>();
```

Reply:
would be nice if github had a replace all